mvc-web/src/main/kotlin/com/example/model/DatabaseFactory.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.model
+
+import com.example.model.dao.wishlist.Wishlists
+import kotlinx.coroutines.Dispatchers
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+import org.jetbrains.exposed.sql.transactions.transaction
+
+object DatabaseFactory {
+    fun init() {
+        val driverClassName = "org.h2.Driver"
```

Comment:
Please move the DB configuration keys to the `application.yaml` or `application.properties`